frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -41,7 +42,6 @@ class UserSuccessModal extends React.Component {
 export default _.compose(
   User.load({
     id: (state, props) => props.params.userId,
-    wrapped: true,
```

Comment:
Since there's no need for user entity methods (`getName`), we can simply pass an object instead.